rancher/charts
# Csp adapter release v2.0.4

## Issue: https://github.com/rancher/rancher/issues/44107

## Problem:
Support for k8s 1.27

## Testing:
scenario 1.
-  Install rancher v2.7.11 and csp-adapter v2.0.4 on an EKS Kubernetes (k8s) 1.27 cluster.
- Validated the support config and licence entitlement functionality. Created a downstream EKS cluster with 21 nodes (with 1 entitlement) to trigger out-of-compliance message. Scaled down the cluster to 17 nodes, validated that the out-of-compliance message is no longer displayed. Scaled back up to 21 nodes, validated the out-of-compliance message popped up.
- uninstall/downgrade/upgrade

scenario 2.
- Install rancher v2.7.11 and csp-adapter v2.0.4 on an EKS Kubernetes (k8s) 1.26 cluster.
- Upgraded EKS cluster to k8s 1.27 in aws
- Validated the support config and licence entitlement functionality. Created a downstream EKS cluster with 21 nodes (with 1 entitlement) to trigger out-of-compliance message. Scaled down the cluster to 17 nodes, validated that the out-of-compliance message is no longer displayed. Scaled back up to 21 nodes, validated the out-of-compliance message popped up.
